components/datetime/src/pattern/names.rs

```diff
@@ -2229,6 +2282,12 @@ impl<FSet: DateTimeNamesMarker> RawDateTimeNames<FSet> {
                 (FS::TimeZone(TimeZone::Location), One) => {
                     // no data required
                 }
+                // VVV
+                (FS::TimeZone(TimeZone::Location), Three) => {
+                    numeric_field = Some(field);
```

Comment:
Question: Do we need a numeric field in this format?

Reply:
no